modules/ocf/manifests/prometheus/node.pp

```diff
@@ -0,0 +1,3 @@
+class ocf::prometheus::node {
+  include prometheus::node_exporter
+}
```

Comment:
This is probably fine, but I feel like just including the node exporter directly in the init manifest would also be fine, you don't have to make this wrapper class if there's nothing to put in it yet.

Reply:
Sure, I'll get rid of this file.